### DISCUSSION TITLE
Tecla de Atalho

### DISCUSSION THREAD

**Discussion by 0m3rpu51q** [elkowar/eww#731] [Q&A]
Num primeiro momento, quero parabenizar todo o grupo pelo trabalho excelente!

à instalaçao do Eww, foi de acordo e sem erros à reportar, entretanto, sua ativação é realizado pelas teclas Shift + S, sendo assim, toda a vez que estamos trabalhando com texto, o respectivo wight (ativa ou desativa), nestas condições, gostaria de saber como faço para alterar às teclas de atalho de sua ativação ou desativação. 

atencisamente, 
agradeço pela atenção, 
e mais uma vez parabéns pelo trabalho.
------------------------------------------------------------------------------------------
At first, I want to congratulate the whole group for the excellent work!

to the installation of Eww, it was ok and without errors to report, however, its activation is carried out by the keys Shift + S, therefore, every time we are working with text, the respective wight (activates or deactivates), in these conditions, I would like to know how do I change the shortcut keys of its activation or deactivation.

Sincerely,
thank you for your attention,
and once again congratulations for the work.

**Comment by elkowar** [comment]
Hey, glad you're enjoying eww!
Eww does not come with any sort of keybinds internally, so I assume the shortcut you're referring to might be part of your distribution, or a set of dotfiles that you have installed externally. Please check with them on how to adjust it -- hint: it's probably in your window manager config!
